ols/utils/auth_dependency.py

```diff
@@ -114,6 +115,16 @@ def get_authz_api(cls) -> kubernetes.client.AuthorizationV1Api:
             cls()
         return cls._authz_api
 
+    @classmethod
+    def get_custom_objects_api(cls) -> kubernetes.client.CustomObjectsApi:
+        """Return the custom objects API instance.
+
+        Ensures the singleton is initialized before returning the Authorization API client.
+        """
+        if cls._instance is None or cls._authz_api is None:
```

Comment:
Should this be `cls._custom_objects_api is None`

Reply:
definitely, thx